scripts/get_torch_nightly_wheels.py

```diff
@@ -0,0 +1,90 @@
+import os
+import re
+import requests
+from datetime import date, timedelta
+from bs4 import BeautifulSoup
+from collections import defaultdict
+
+torch_wheel_nightly_base ="https://download.pytorch.org/whl/nightly/cu102/" 
+torch_nightly_wheel_index = "https://download.pytorch.org/whl/nightly/cu102/torch_nightly.html" 
+
+def memoize(function):
+    """ 
+    """
+    call_cache = {}
+
+    def memoized_function(*f_args):
+        if f_args in call_cache:
+            return call_cache[f_args]
+        call_cache[f_args] = result = function(*f_args)
+        return result
+
+    return memoized_function
+
+@memoize
+def get_wheel_index_data(py_version, platform_version, url=torch_nightly_wheel_index):
+    """
+    """
+    r = requests.get(url)
+    r.raise_for_status()
+    soup = BeautifulSoup(r.text, 'html.parser')
+    links = soup.find_all('a')
+    data = defaultdict(dict)
+    for link in soup.find_all('a'):
+        pkg, version, py, py_m, platform = re.search("([a-z]*)-(.*)-(.*)-(.*)-(.*)\.whl", link.text).groups()
```

Comment:
Currently the link.text on torch_nightly.html is broken, maybe it's better to use link['href']?